pages/category/index.vue

```diff
@@ -141,14 +141,16 @@
             :show-add-to-cart-button="true"
             :score-rating="3"
             :max-rating="5"
-            wishlist-icon=""
-            is-in-wishlist-icon=""
-            :is-in-wishlist="false"
+            wishlist-icon="heart"
+            is-in-wishlist-icon="heart_fill"
+            :is-in-wishlist="isInWishlist(product)"
             :regular-price="productGetters.getPrice(product).regular"
             :special-price="productGetters.getPrice(product).special"
```

Comment:
Change this to 
:special-price="
              productGetters.getPrice(product).special &&
              $n(productGetters.getPrice(product).special, 'currency')
            "

Reply:
Done.